app/templates/index.html

```diff
@@ -12,6 +12,8 @@
           {% set displayed_message = "Could not find a service with ID: {}".format(message['no_service']) %}
         {% elif message['api_error'] %}
           {% set displayed_message = "Error trying to retrieve service with ID: {}".format(message['api_error']) %}
+        {% elif message['api_no_frameworks'] %}
+          {% set displayed_message = "Could not retrieve frameworks." %}
```

Comment:
I don't think this is worth flashing a message about - if we can't retrieve any frameworks then either the API is down and the page should probably just break, or there are no "standstill" or "live" frameworks and we all lost our jobs several months previously.